lib/src/painters/solid_path_painter.dart

```diff
@@ -3,6 +3,7 @@ import 'package:flutter/material.dart';
 /// Draw a solid path around the given path
 class SolidCropPathPainter extends CustomPainter {
   static const _strokeWidth = 4.0;
+  Color outlineColor;
```

Comment:
This can be final. I would also make this optional with Colors.white as fallback

Reply:
I would not use white, but transparant because it would break/update all current implementation to an outline implementation

Reply:
The current implementation uses white
```
  final _paint = Paint()
    ..color = Colors.white
	....
  void paint(Canvas canvas, Size size) {
    _paint.color = outlineColor;
...
```

Reply:
forget what I said here :D